### PR TITLE
Assurance bar styling for money

### PR DESCRIPTION
**Pull request by hcanadij**
# Description

Adding Assurance Bar Card variation styling to money theme

image

# Checklist

Pull request contains:

- [ ] A new component
- [X] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [X] Work has been tested in multiple browsers
- [X] PR description includes description of change
- [X] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/themes/money/theme.json [hunk 1]
```diff
@@ -3292,6 +3292,59 @@
...
+          "flexDirection": "row",
+          "flexWrap": "wrap",
+          "alignItems": "flex-start",
+          "textAlign": "left",
+          "py": 0,
+          "mb": "md",
+          "h3": {
+            "color": "#000",
+            "paddingLeft": "0px",
+            "fontFamily": "Nunito, sans-serif",
```
Comment (phelma) [inline src/themes/money/theme.json:3307]: this should be "fontFamily": "heading"

File: src/themes/money/theme.json [hunk 2]
```diff
@@ -3292,6 +3292,59 @@
...
+        "assuranceBar": {
+          "display": "flex",
+          "flexDirection": "row",
+          "flexWrap": "wrap",
+          "alignItems": "flex-start",
+          "textAlign": "left",
+          "py": 0,
+          "mb": "md",
+          "h3": {
+            "color": "#000",
```
Comment (phelma) [inline src/themes/money/theme.json:3305]: should be `grey-100`

File: src/themes/money/theme.json [hunk 3]
```diff
@@ -3292,6 +3292,59 @@
...
+            "paddingLeft": "0px",
+            "fontFamily": "Nunito, sans-serif",
+            "fontWeight": 600,
+            "fontSize": 30,
+            "lineHeight": "120%"
```
Comment (phelma) [inline src/themes/money/theme.json:3310]: should be "lineHeight": "heading"

File: src/themes/money/theme.json [hunk 4]
```diff
@@ -3292,6 +3292,59 @@
...
+          "flexWrap": "wrap",
+          "alignItems": "flex-start",
+          "textAlign": "left",
+          "py": 0,
+          "mb": "md",
+          "h3": {
+            "color": "#000",
+            "paddingLeft": "0px",
+            "fontFamily": "Nunito, sans-serif",
+            "fontWeight": 600,
```
Comment (phelma) [inline src/themes/money/theme.json:3308]: should be "fontWeight": "bold"

File: src/themes/money/theme.json [hunk 5]
```diff
@@ -3292,6 +3292,59 @@
...
+            "fontSize": 30,
```
Comment (phelma) [inline src/themes/money/theme.json:3309]: should be fontSize: "xxl"

File: src/themes/money/theme.json [hunk 6]
```diff
@@ -3292,6 +3292,59 @@
...
+            "lineHeight": "120%"
+          },
+          "p": {
+            "color": "#000",
+            "mx": 0,
+            "fontFamily": "SF Pro Text",
```
Comment (phelma) [inline src/themes/money/theme.json:3315]: not sure where this font has come from, but it should not be used on money
should be fontFamily: "base"

**Review by phelma — CHANGES_REQUESTED** [review]
we should be using the tokens from the theme and not setting values like this everywhere

also if they are just normal heading styles then we can just use a variant

File: src/themes/money/theme.json (hunk 6 above)
Comment (hcanadij) [reply src/themes/money/theme.json:3315]: It's the font used in designs :/

**Comment by hcanadij** [comment]
Updated with token values

**Review by phelma — APPROVED** [review]

**Pull request merged**